moto/panorama/models.py

```diff
@@ -0,0 +1,304 @@
+        self.__device_aggregated_status_manager.advance()
+        return _device_aggregated_status
+
+    @property
+    def provisioning_status(self) -> PROVISIONING_STATUS_TYPE:
+        _provisioning_status: PROVISIONING_STATUS_TYPE = self.__device_provisioning_status_manager.status  # type: ignore[assignment]
+        self.__device_provisioning_status_manager.advance()
+        return _provisioning_status
+
+    def response_object(self) -> Dict[str, Any]:
+        response_object = super().gen_response_object()
+        response_object = deep_convert_datetime_to_isoformat(response_object)
+        static_response_fields = [
+            "AlternateSoftwares",
+            "Arn",
+            "Brand",
+            "CreatedTime",
+            "CurrentNetworkingStatus",
+            "CurrentSoftware",
+            "Description",
+            "DeviceConnectionStatus",
+            "DeviceId",
+            "LatestAlternateSoftware",
+            "LatestDeviceJob",
+            "LatestSoftware",
+            "LeaseExpirationTime",
+            "Name",
+            "NetworkConfiguration",
+            "SerialNumber",
+            "Tags",
+            "Type",
+        ]
+        return {
+            **{
+                k: v
+                for k, v in response_object.items()
+                if v is not None and k in static_response_fields
+            },
+            **{
+                "DeviceAggregatedStatus": self.device_aggregated_status,
+                "ProvisioningStatus": self.provisioning_status,
+            },
+        }
+
+    def response_listed(self) -> Dict[str, Any]:
+        response_object = super().gen_response_object()
+        response_object = deep_convert_datetime_to_isoformat(response_object)
+        static_response_fields = [
+            "Brand",
+            "CreatedTime",
+            "CurrentSoftware",
+            "Description",
+            "DeviceId",
+            "LastUpdatedTime",
+            "LatestDeviceJob",
+            "LeaseExpirationTime",
+            "Name",
+            "Tags",
+            "Type",
+        ]
+        return {
+            **{
+                k: v
+                for k, v in response_object.items()
+                if v is not None and k in static_response_fields
+            },
+            **{
+                "DeviceAggregatedStatus": self.device_aggregated_status,
+                "ProvisioningStatus": self.provisioning_status,
+            },
+        }
+
+    @property
+    def response_provision(self) -> Dict[str, Union[str, bytes]]:
+        return {
+            "Arn": self.arn,
+            "Certificates": base64.b64encode("certificate".encode("utf-8")).decode(
+                "utf-8"
+            ),
+            "DeviceId": self.device_id,
+            "IotThingName": self.iot_thing_name,
+            "Status": "AWAITING_PROVISIONING",
```

Comment:
Can we use `self.__device_provisioning_status_manager.status` here, instead of hardcoding the status?

Reply:
Done: https://github.com/getmoto/moto/pull/6948/commits/90828eb8248b14b311cfe55e11c01c4b71c8368f